slider_joint_1_position_controller:
  type: effort_controllers/JointPositionController
  joint: slider_joint_1
  pid: {p: 100.0, i: 0.01, d: 10.0}

slider_joint_2_position_controller:
  type: effort_controllers/JointPositionController
  joint: slider_joint_2
  pid: {p: 100.0, i: 0.01, d: 10.0}


slider_joint_3_position_controller:
  type: effort_controllers/JointPositionController
  joint: slider_joint_3
  pid: {p: 100.0, i: 0.01, d: 10.0}

slider_joint_4_position_controller:
  type: effort_controllers/JointPositionController
  joint: slider_joint_4
  pid: {p: 100.0, i: 0.01, d: 10.0}
